certora/confs/ProxySimulator.conf

```diff
@@ -0,0 +1,35 @@
+{
+    "assert_autofinder_success": true,
+    "files": [
+        "modules/passkey/contracts/ProxySimulator.sol",
+        "modules/passkey/contracts/SafeWebAuthnSignerProxy.sol",
+        "modules/passkey/contracts/SafeWebAuthnSignerSingleton.sol",
+        "modules/passkey/contracts/libraries/P256.sol",
+    ],
+    "link": [
+        "ProxySimulator:_proxy=SafeWebAuthnSignerProxy",
+        "SafeWebAuthnSignerProxy:_VERIFIERS=P256",
+        "SafeWebAuthnSignerProxy:_SINGLETON=SafeWebAuthnSignerSingleton"
+    ],
+    "java_args": [
+        " -ea -Dlevel.setup.helpers=info"
+    ],
+    "msg": "sanity_with_all_default_summaries",
+    "process": "emv",
+    "prover_args": [
+        " -verifyCache -verifyTACDumps -testMode -checkRuleDigest -callTraceHardFail on"
```

Comment:
I think we don't need those and also other, java_args.
We need to prepare for passing to safe and will have it in their ci. we can look at other configs they have

Reply:
Agreed. removed.